### ISSUE TITLE
Add ability to generate multiple placeholder images

### ISSUE THREAD

**Issue by ZachTRice** [bootpack/bootpack#57]
The ability to output an image 1 at a time is a bit limiting, sometimes it would be nice to define a set (array?) or image sizes to be output all at once. This might work best as a separate file (JSON? comma delimited list?).

Example
```json
{
    "images": [
        {
            "title": "banner",
            "width": 1200, 
            "height": 320, 
            "quality": 100
        },
        {
            "title": "thumbnail",
            "width": 80, 
            "height": 80, 
            "quality": 100
        }
    ]
}

**Comment by ZachTRice** [comment]
Another (simpler) approach would be to modify the current `npm run generate:images...` command to append the file width & height dimensions + a `_###` value to the output image name.

Example:
- Run `npm run generate:images width 200 height 200`; creates: `placeholder-200x200_001.jpg`
- Run the exact same command again; creates: `placeholder-200x200_002.jpg`
- Run `npm run generate:images width 300 height 300`; creates: `placeholder-300x300_001.jpg`

Either approach would be acceptable. Either approach will likely need to append and increment a `_###` value onto the filename in order to avoid conflict with multiple files of the same name.

**Comment by ZachTRice** [comment]
Addressed in #168
